django_celery_beat/models.py

```diff
@@ -57,7 +57,8 @@ def crontab_schedule_celery_timezone():
     except AttributeError:
         return 'UTC'
     return CELERY_TIMEZONE if CELERY_TIMEZONE in [
-        choice[0].zone for choice in timezone_field.TimeZoneField.CHOICES
+        choice[0].zone for choice in timezone_field.
```

Comment:
I guess this also need change in model migrations?

Reply:
@auvipy I don't think any other changes will be needed, the tuples outputted by `TimeZoneField.CHOICES` and `default_choices` are the same and so the data should be the same.

Reply:
```
(venv) ~/D/django-celery-beat ❯❯❯ python3.9 manage.py makemigrations                                                                         master
No changes detected
```